cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -72,6 +72,51 @@ message DeleteSurveyResponse {
     bool success = 1;
 }
 
+/**
+    Source Files
+**/
+
+message RegisterSourceSegyFileRequest {
+    Identifier survey = 1;                              // [required] The survey this source file belongs to
+    string path = 2;                                    // [required if file is not synthetic] Path including protocol, bucket and directory structure. Example: "gs://cognite-seismic-eu/samples/"
+    string name = 3;                                    // [required] Unique filename including extension. Example: "DN1302M03R16_MERGED_KPSDM_00-32_DEG_T.sgy". The name must be unique across buckets and can be used to identify this file in query requests
+    ExternalId external_id = 4;                         // [optional] An external identifier - matches service contract field
+    map<string, string> metadata = 5;                   // [optional] Any custom-defined metadata
+    CRS crs = 6;                                        // [required] Official name of the CRS used. Example: "EPSG:23031"
+    google.protobuf.BoolValue is_temporary = 7;         // [optional] Tells whether file is temporary (writeable) or not. False by default
```

Comment:
I've always wondered why this is here. For eventual writeback support? Should it maybe be delayed?

Reply:
Aye, let's omit this in v1 for now.

Reply:
Backstory: We implemented a PoC for AkerBP letting them write trace data from processing jobs. At that time we didn't have the separation of concerns between files and seismicstores, so we suddenly had to have files that didn't exist in a backing store, weren't ingested, et cetera. This was considered a massive hack. This is one impetus for the wish to keep files and seismic stores as separate entities, with semi-independent life cycles. =)

Reply:
(So what we did was to create a special case of file with this is_temporary flag, that wouldn't be ingested, and an API for writing traces straight into Bigtable.)

Reply:
Right, so we should forego it here indeed.